cmd/dcrctl/version.go

```diff
@@ -27,9 +27,9 @@ const (
 )
 
 // appBuild is defined as a variable so it can be overridden during the build
-// process with '-ldflags "-X main.appBuild foo' if needed.  It MUST only
+// process with '-ldflags "-X main.appBuild=foo' if needed.  It MUST only
 // contain characters from semanticAlphabet per the semantic versioning spec.
-var appBuild string
+var appBuild = "dev"
```

Comment:
Nope.  As far as I can tell, const does not work with the ldflags trick.  See this code:
```
package main

import "fmt"

const (
	test = "hi"
)

func main() {
	fmt.Println(test)
}
```

```
jcv@triforce tmp $ go run ldflag.go 
ho
jcv@triforce tmp $ go run -ldflags "-X main.test=ho" ldflag.go 
ho
```

Changing it to a var and it works as expected.